code/game/machinery/bees_apiary.dm

```diff
@@ -12,8 +12,7 @@
 	var/mut = 1
 	var/toxic = 0
 	var/dead = 0
-	var/health = -1
-	var/maxhealth = 100
+	max_integrity = 100
```

Comment:
This is not really a maintained feature, but to preserve behavior it should have `obj_integrity = -1` by default.

Reply:
gross

Reply:
It is how it is. A quick glance shows me that this tracks the work of bees and production, not just object integrity. As it is the change will break this.

Reply:
fuck bees